README.md

```diff
@@ -0,0 +1,68 @@
+# Apple Income Statements
+
+This is a project developed in React and Rudux, that uses the [Financial Modeling Prep API](https://site.financialmodelingprep.com/developer/docs) to create a single page application that display each statement of the last 5 years. The design was inspired by Nelson Sakwa's [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
+
+
+## Screenshot
+
+![screenshot](./public/apple.png)![screenshot](./public/apple1.png)
+
+
+## Built With
+
+- HTML 
+- CSS 
+- JavaScript
+- React
+- Redux
+
+## Live Demo
+
+[GitHub Link](https://fabianofrank.github.io/apple-income/)
```

Comment:
- [ ] In this section of the README, we should have the following:
  - A link to the project's video presentation
  - A link to the deployed live version of the project
  Please add these to the README.
  
  ### Answer to the question you left in the Code Review Request
  I would recommend you use Netlify for deployment. It is a simpler process and this React would be very responsive on Netlify. Here is a [GUIDE](https://medium.com/geekculture/how-to-deploy-react-projects-on-netlify-28f72b4ceb8c) to help with the deployment to Netlify.